client/src/redux/app-mount-saga.js

```diff
@@ -2,23 +2,31 @@ import qs from 'query-string';
 import { put, takeEvery } from 'redux-saga/effects';
 import { createFlashMessage } from '../components/Flash/redux';
 
+function* clearSearchWithoutReRender() {
+  // Clear the search without causing a re-render
+  const newUrl = window.location.pathname;
+  window.history.pushState({}, '', newUrl);
+}
+
 function* parseMessagesSaga() {
   const search = window.location.search.slice();
-  // TODO(Bouncey): Find a way to clear the search with causing a re-render
+  // TODO: (Bouncey) Find a way to clear the search without causing a re-render
```

Comment:
Is this still a todo? Is this not what the PR changes are accomplishing?

Reply:
Hi, the TODO comments can be removed.

Reply:
```suggestion
```